worldedit-core/src/main/java/com/sk89q/worldedit/command/SchematicCommands.java

```diff
@@ -355,19 +383,18 @@ public void save(Actor actor, LocalSession session,
         if (parent != null && !parent.exists()) {
             if (!parent.mkdirs()) {
                 throw new StopExecutionException(TranslatableComponent.of(
-                        "worldedit.schematic.save.failed-directory"));
+                    "worldedit.schematic.save.failed-directory"));
             }
         }
 
         ClipboardHolder holder = session.getClipboard();
 
         SchematicSaveTask task = new SchematicSaveTask(actor, f, format, holder, overwrite);
         AsyncCommandBuilder.wrap(task, actor)
-                .registerWithSupervisor(worldEdit.getSupervisor(), "Saving schematic " + filename)
-                .sendMessageAfterDelay(TranslatableComponent.of("worldedit.schematic.save.saving"))
-                .onSuccess(filename + " saved" + (overwrite ? " (overwriting previous file)." : "."), null)
```

Comment:
What happened to the onSuccess part?

Reply:
> What happened to the onSuccess part?

originally the player would get two notifications for a successful save, one coming from where you pointed out,
`        AsyncCommandBuilder.wrap(task, actor)
            .registerWithSupervisor(worldEdit.getSupervisor(), "Saving schematic " + filename)
            .sendMessageAfterDelay(TranslatableComponent.of("worldedit.schematic.save.saving"))
            .onFailure("Failed to save schematic", worldEdit.getPlatformManager().getPlatformCommandManager().getExceptionConverter())
            .buildAndExec(worldEdit.getExecutorService());
`
and the other coming from
`log.info(actor.getName() + " saved " + file.getCanonicalPath());
 actor.print(Caption.of("fawe.worldedit.schematic.schematic.saved", SCHEMATIC_NAME));`


I thought telling the player they had succeeded two times was redundant, so I dropped one of them - though looking back, maybe I should have removed the second one instead. Should I do that?

Reply:
Yeah it's probably better to remove the second one but keep the log output

Reply:
will do!